src/casync-tool.c

```diff
@@ -1795,6 +1797,8 @@ static int list_one_file(const char *arg0, CaSync *s, bool *toplevel_shown) {
                 (void) ca_sync_current_fat_attrs(s, &fat_attrs);
                 (void) ca_sync_current_archive_offset(s, &offset);
 
+                has_quota_projid = ca_sync_current_quota_projid(s, &quota_projid) >= 0;
```

Comment:
But this is the "stat" command, i.e. the call which is supposed to show you the full monty, everything we got. The "mtree" and "list" commands won't show this stuff, for a good reason.

Moreover this is only shown if the quota projid stuff is actually enabled. It's a feature that most people shouldn't enable I guess. at least not the folks who want to backup their home directory, and not the folks who distribute iot images with this. The only folks who should enable that are people doing "full system backups"... Now you might argue that people are likely to run things with default feature flags on, and that means with quota projid enabled. But to that I'd respond: if we then show the field it's a good thing, so that people realize that their feature flags are a bit too broadly set, and that this the prompts them to revisit their choices on that...

In general, I think in casync we should focus on reproducibility, and that means that the same settings+data result in the same output and different settings or different data in different output. Hence I think it's a good thing if fields in the "stat" output appear depending only on the selected feature flags — but not depending on the actual data values: that way output will differ if you run "casync stat" once with the quota-projid feature flag on and once with it off, even if the projid id actually set on files is 0...

Oh, and the way this is currently is implemented we actually 'support' the quota ID on btrfs too: we pretend that all files/directories on btrfs had the quota id 0. That means you can extract images with quota ID enabled on btrfs too, as long as the quota id is never set to anything other than 0... if you run "casync stat ." on btrfs you'll see the quota id as 0...

Hope the above makes some sense?